City-Bureau/city-scrapers-wichita
# 🏗️  Build boarddocs mixin and 3 spiders

## What's this PR do?
Adds a mixin for scraping Board Docs websites and adds 3 spiders, including:
- Goddard Board of Education: `wicks_goddard_boe`
- Maize Unified School District Board: `wicks_maize_usd`
- Andover Board of Education: `wicks_andover_boe`

## Why are we doing this?
Wichita is a new partner in the network and needs scrapers.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl wicks_goddard_boe -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here]().

## Are there any smells or added technical debt to note?
- As with some other recent mixin-related PRs, I've only written a test for one of the spiders that use this mixin. Writing more seems redundant. 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206670730876983
  - https://app.asana.com/0/0/1206607687981276
